src/Materials/effectRenderer.ts

```diff
@@ -128,68 +94,47 @@ export class EffectRenderer {
         this.engine.drawElementsType(Constants.MATERIAL_TriangleFillMode, 0, 6);
     }
 
+    private isRenderTargetTexture(texture: InternalTexture | RenderTargetTexture): texture is RenderTargetTexture  {

```

Comment:
Love this one :-)

Reply:
Yes, avoid a not very readable line in the render code!